ddtrace/tracer.py

```diff
@@ -7,6 +7,7 @@
 from .writer import AgentWriter
 from .span import Span
 from . import compat
+from os import getpid
```

Comment:
Just to be sure, we have some benchmarks about the impact of creating a trace. Can you double check using existing (or writing new one in `tests/benchmark.py`) benchmarks so that we're sure that there is no impact in making those system calls? We already do that once, and it wasn't as far as I remember. But just to be sure.